docs/build/web.md

```diff
@@ -60,10 +62,22 @@ There are 2 steps to build ONNX Runtime Web:
 
 ### Build Instructions
 
+ONNX Runtime WebAssembly can be built with or without multi-thread and Single Instruction Multiple Data (SIMD) support.
+This support is added/removed by appending the following flags to the build command, the default build option is without.
+
+| build flag              | usage                           |
+| ----------------------- | ------------------------------- |
+| `--enable_wasm_threads` | build with multi-thread support |
+| `--enable_wasm_simd`    | build with SIMD support         |
+
+A complete build for ONNX runtime WebAssembly artifacts will contain 4 ".wasm" files, covering all ON/OFF configuration of these flags.
+The build command below should be run for each of the configurations.
+
 in `<ORT_ROOT>/`, run one of the following commands to build WebAssembly:
 
 ```sh
 # In windows, use 'build' to replace './build.sh'
+# It's recommended to use '--skip_tests` due to performance and memory hogging concerns - please review FAQ for more details
```

Comment:
It's a good explanation. but to be more accurate, we do "skip_tests" only for release build. this is because:

- for debug build, we don't care the perf anyway; instead we want to make sure the correctness is validated
- for release build, we CANNOT run tests because we disabled exception catching, and tests require exception to work.

Reply:
Got it. I update the recommendation with this added info :)